### DISCUSSION TITLE
How to change effect dynamically using swiper/vue?

### DISCUSSION THREAD

**Discussion by lucassimines** [nolimits4web/swiper#6561] [General]
I need to change the `effect` based on window size.

I'm using the <Swiper> component and Vue3, but I can't make the effect change, even when the variable has changed from 'fade' to 'slide'...

How can I do that?

**Comment by geeksalah7** [comment]
I was facing the same problem, here is a work around.
Add a` v-if` to the swiper slider 
`<swiper  v-if="showSlider" :effect="store.config.layout.slider.effect"></swiper>`
In my setup hook 
`let showSlider = ref(true)
watch(()=> store.config.layout.slider.effect, ()=>{
  showSlider.value = false
  setTimeout(()=> showSlider.value = true, 600)
})`
